subprojects/core/src/main/java/org/gradle/api/internal/tasks/NonCacheableTaskOutputPropertySpec.java

```diff
@@ -48,8 +48,8 @@ public String getPropertyName() {
     }
 
     @Override
-    public SnapshotNormalizationStrategy getSnapshotNormalizationStrategy() {
-        return parent.getSnapshotNormalizationStrategy();
+    public PathNormalizationStrategy getPathNormalizationStrategy() {
+        return parent.getPathNormalizationStrategy();
     }
```

Comment:
This doesn't seem to be necessary anymore. We can simply return the output normalization strategy here.